Packs/Traceable/Integrations/Traceable/Traceable.py

```diff
@@ -809,26 +944,15 @@ def get_threat_events(
                 domain_event["apiType"] = "Unknown"
 
             if ("id" in domain_event and "value" in domain_event["id"] and self.app_url != ""
-                    and "environment" in domain_event and "value" in domain_event["environment"]):
+                    and "environment" in domain_event and domain_event["environment"] is not None
+                    and domain_event["environment"] != ""):
```

Comment:
Please use the dict `get` method
https://www.w3schools.com/python/ref_dictionary_get.asp

Reply:
Done